src/main/webapp/js/item_page_loader.js

```diff
@@ -15,10 +19,16 @@ function createMap(user_lat, user_lng, user_title) {
 }
 
 function buildUI() {
+  // URL must include ?user=XYZ parameter. If not, redirect to homepage.

```

Comment:
So if you're using the user to show their posting, what happens when they want to have multiple postings? 
Also this gives away information in the URL about the poster (probably fine in this case).

This is fine for now, but keep in mind you'll have to adjust things if you want to scale :) One posting per user is a good starting point though

Reply:
Yes for sure. This is already considered in a TO-DO. A lot of the code is based on one posting per user (for example location is fetched from user. It is not in the actual item). Will rework if we have time.